swap_meet/item.py

```diff
@@ -1,2 +1,34 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self,id=None,condition=0, age=0):
+        if id is not None and not isinstance(id, int):
+            raise TypeError ("id must be an integer")
+        self.id = uuid.uuid4().int if id is None else id
```

Comment:
I would add a space between these two lines of code so you can better visualize the difference in responsibility between the two. One section's responsibility is data validation and the other section is attribute declaration. 
```suggestion
        if id is not None and not isinstance(id, int):
            raise TypeError ("id must be an integer")
            
        self.id = uuid.uuid4().int if id is None else id
```